modules.d/99base/module-setup.sh

```diff
@@ -67,29 +67,24 @@ install() {
         echo ro >> "${initdir}/etc/cmdline.d/base.conf"
     fi
 
+    [ -e "${initdir}/usr/lib" ] || mkdir -m 0755 -p ${initdir}/usr/lib
+
     local VERSION=""
     local PRETTY_NAME=""
     if [ -e /etc/os-release ]; then
         . /etc/os-release
+        grep -hE -ve '^VERSION=' -ve '^PRETTY_NAME' /etc/os-release >${initdir}/usr/lib/os-release
         [[ -n ${VERSION} ]] && VERSION+=" "
         [[ -n ${PRETTY_NAME} ]] && PRETTY_NAME+=" "
     fi
-    NAME=dracut
-    ID=dracut
```

Comment:
Better move that up before the `. /etc/os-release` in case it does not exist, so have a default value.

Reply:
Or, just hard fail if the system doesn't have an `/etc/os-release`?

Reply:
> Or, just hard fail if the system doesn't have an `/usr/lib/os-release` (or `/etc/os-release`)?

Nah, as we learned, there are quite a few distributions using dracut without systemd. *BSD even.

Reply:
:scream: :cry: 
Feels like it'd make sense for them to fork and have one dracut focused solely on systemd.  But OK not for this PR...